src/lib/replication/repl_dev/raft_repl_dev.cpp

```diff
@@ -594,7 +594,8 @@ folly::Future< folly::Unit > RaftReplDev::notify_after_data_written(std::vector<
     });
 }
 
-bool RaftReplDev::wait_for_data_receive(std::vector< repl_req_ptr_t > const& rreqs, uint64_t timeout_ms) {
+bool RaftReplDev::wait_for_data_receive(std::vector< repl_req_ptr_t > const& rreqs, uint64_t timeout_ms,
```

Comment:
1 we have to places to call `wait_for_data_receive`, in `handle_raft_event` and  `end_of_append_batch` . now that we have guaranteed the data is received in handle_raft_event, can we remove wait_for_data_receive in notify_after_data_written(end_of_append_batch)? since there is no timeout_rreqs in that case.

Reply:
we can remove this based on no pack/unpack.